docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1783 @@
+{
```

Comment:
<p>@<a href="https://github.com/ricardoV94" target="_blank">ricardoV94</a> Maybe you can help me with a few sentences for this intro! You have a much more in-depth knowledge than me to choose the right words :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
<p>What about something along these lines?</p><ol><li>Logp graph does not have random variables (obviously)</li><li>Instead it's defined in terms of input (value) variables</li><li>Each RV is replaced by a respective input (value) variable</li></ol><p> </p>

Reply:
Added these comments in https://github.com/pymc-devs/pymc/pull/5721/commits/b0781cf835d30257a8458bac80c76db3556936a6

Reply:
<p>Added these comments in https://github.com/pymc-devs/pymc/pull/5721/commits/b0781cf835d30257a8458bac80c76db3556936a6</p>